infrastructure/cdn-in-a-box/optional/grafana/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM grafana/grafana:5.4.3
+
+USER root
+ARG TRAFFIC_TS_RPM=traffic_stats/traffic_stats.rpm
+
+RUN apt-get update && \
+    apt-get install -y dnsutils net-tools vim tree procps gettext-base curl p7zip-full sudo libcap2-bin netcat && \
```

Comment:
Does this really need `vim`, `tree`, `sudo` and whatever `procps` is? In fact, I'd go so far as to say I *know* it doesn't need `vim`. I know you're using `sudo` to execute `/run.sh`, but surely you could use `su` instead? Especially because it's used at the "exit point" of `run-grafana.sh`, most of the features of `sudo` are wasted on such a simple use case.